README.md

```diff
@@ -17,30 +17,31 @@ Se você quiser contribuir porfavor sinta-se livre :heart:.
/iMusWHlvardiaS5+NVioDXVG9ZGeiArv4Sctv25jGvI+P9bbYhxntRIYtt2KZQXkpUTUomsb7qiqBmGBwNqwGlYSq//AHu6+lsuM+gKP/yEr946nIZt9fyx1JuHfc1Wbsf1vaN4/LdYo6s9dnYdLLaHxlvIyU2jHBt5ahu9alva0DBaW9h6+H6ysxlbEuv4nz5achBd9I/ZpM4rv2imNvCV58YZ5YWEX74m7p58Ty7iv3R3apVOuz1VCk28/+6liO/fgX9m0M2R/GUWogka8TfycWNH6/1Dn0ZL7HBpvOONv07uene8N1adsEM+wk3vEz4l1lOxzZTrcTN1Tm57YXKZk2f1GOquW9ye1n/6cWFfs0b7XebTEZpJ5C1c84STpwO4h6qAukmoFr+lIzefEumOPViswOeLSnEbSxB5qUvZ9hGjUBu+3x1fB7YgV9buO825wYeJwZvOepyWcrf3Es6yWRNKkdzUX0tq9WL+4LmdxYmK0xYl7Cv5p1sr3n//6KzuutjUNhSEYMbGL9ZhMjvfj8ZTeW6RUfdOWkmc3tjsNo56ficuKehPvc1JhhjgSeMFDzPsSS7VsupUeBK/aFsXq1H60Ip8q4Xv9ipL2NQ0XPEgstaxKYmlTccNBUqx+1J2+uHCOOFR4n41y5Z2tPRn4l+RRapA858mJ1WZiqC/ihlusskLljyZVRaOV2Ofkdt6JyBibiHsRK9pO87zXku0O0lmOIjy/l15Ztk6z2THot9/VhtY5F0Tg88czlJnAfTilas769EpGFpowJVl8J4Mb6Tyj04gPh0o+/DiKfuVjVygV4wpt/x62O414hHhTJo1cHHVAfpSn+flmp5HBs+wwgXewihOf2+ekiXc9oVex4cBaxOfEsmbcXHWfrbcooLUsjGchKZqHXJAYXq2hsHpxFG+zWne8bMHthXjXQ896+GrwS1xcRCTWfZ0oNxPLWkUJg/c3MoOs8FCsOFKaGFoNdZK6V+St4gHpsrRxItC54fn40ll4eWEc73rITiietdWdXr4/uK66QCMeKX3n7yP/SgvVMhZqkVjSnNt/tdEyWYG23NLjdeFVikonzuAvrm/PCyBPfsMLCs4IQ0a8CJlGibXz5Oz7Js6P0i38vtlY5ZZdDQtPdyKrI7mxK8oE7BUcWJAsPhGxXMrrn5bjuM9PSsLhSHisoqOHHCS6ntjnFH7oRmzVRukPogpkkRkX3xIT8aPPxVKZf9WdJR3R8MwHvahFYqFpystXVU1LX8HRkr6I4FArB56gRkGEL02863nAzla/Xs/pQpA7jyznlIvV0GZkKUyGkSkk0ZgVjiysVuntLjVt+Dd//iRuFVv30qtTcWPETlW5R5HgJ6p0nSDFEUrE4GVuy+PO4hWyvdBKxMIzUXibi9bWg0zhs00PdJkqVG55f0fFiNtqnAjbK4S4/qN6VUYo6Wct8nEZy28viq4cMdorSyyXNcl79pvUSom1wWk6SlZC/XtxiwX0K312Ke5QPkSTgW94isTCLQzzcqnW3BSVHZU/0KMRm7aNF0QjoxP6bf06Y4vtoWpN3uZmsTjFYuG52NI1LXYjNE1fFcRtUPOm9ImIYQgXuvuim8kcthLwo4vzh7KWU0QT/Ki6wtSD24ZY2OWQ7dVcVzD6fGWbZWU7t9fFU4ZYeOMMr23XFa5EjInwN7Fla45uHissiw+XcdvG8dlplQl8Gg5fWa+GUioWYSB5GCl7X15Af9wpgg6TZqfiHqTZyYy/fnMzmXXDqFKHjV5Dqki1xwcAAAAAAPgSKvhj/x1k1yG4bvabqmjgFjuwflRa+ijzcp2NLvBh4U2nwcckmE5z/RoMC74JjOH7a2lSpkdJCweL44pSy8Qa8mBE6TQUhHcOHi5Wz2SbZ0OWux7tqmea4b0J/MKUo85zsQx5QPMCs+t1wxeshZ7ck2gLbdmUwy984Ey54Jt6hwMTq60PHVdxEDmUXbmtITmxdxXXmei4/77u2LYTBU25WNKc5CmO7WoKbsV3bUcnrXX1iRO2sNRazkTBQga6S77Fv4f+bRUm1or8aisyMtZElAEeFz0apbAnCD2lQluxWMM2GljE3MsKSzJ0XPSDnnHgFjyaOZ3jOgX3dA6MUKzBmk4R10eSTt9jm+XTQ1tPM8rE8hQ6FelPZBjItVF7TsMbWCxzTYvhMTbPxvIPk1Cs9ppapKWDvDl5QcSy6aGjhHVw5o4fcPciJVbbIspQsaaTj4+PJxt5Qxo6wmJNP2hb6y5aPrn+9OBNViWxkOy7Kz06xU2JhYZut9tb4UHYXcvttvSSEoueiBOxUA+3sBbe1TkkQrGkNR04toNlIy+oWPSKW9tiDgKbalmxevpkPiT3MpdkdUDvZBrSglisIBR+zdZRaXOo9l8OM/ATapTXJhrQm2vSHFscnXTa1FnBgc7E+ojOqKlYZjRcAtKQN7ex0NTEYbm7VOieHp1HiO/MHRBO2Nme4pvmMxkTtmKatk76NX82TV/DlkbGbpa5mrAa5lNgBkEgh67DlB5xYKfA05fmSpnZ5EbAVPZ1DbfwMTfNpRJQJ840Xb34MQ4Dk00qz7ft8GXPdqbektht07Z9MhhM37ffp/wWrUz/89EUDaYDJK9JcnuOR5tnv8uoR0ZV13b9Nm+B+hakheX/auMtwAnNvn/w/uYu8J/JyBrM/htO5xczGM5bbmv+8X+fYdXAm2ZZ7tX8j2oAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA1+BskRugI/uuP8gAAAABJRU5ErkJggg==) | UCALL | Ucall is the largest Angolan service provider in the fields of customer service and experience. | https://www.ucall.co.ao/ | C#, Javascript, PHP, SQL , NodeJs (probably) | Luanda-Angola |
```

Comment:
We could think on another workaround for those images, base64 links are too long